### PR TITLE
Save the particle sizes in the info.xml file

### PR DESCRIPTION
**Pull request by markdewing**
It's useful to know the problem size when analyzing the performance.

### CODE REVIEW

File: src/Particle/ParticleSet.cpp [hunk 1]
```diff
@@ -313,6 +313,13 @@ bool ParticleSet::get(std::ostream& os) const
   return true;
 }
 
+void ParticleSet::output_particle_info(Libxml2Document &doc, xmlNodePtr root)
```
Comment (ye-luo) [inline src/Particle/ParticleSet.cpp]: Can we avoid adding this routine in ParticleSet? I'd like to keep the class as light as possible.
The number of particles and name can be accessed via getName() and getTotalNum()

**Comment by prckent** [comment]
Agreed. Ideally we would employ a design where these core classes know nothing about I/O. Input is of course the biggest headache.

**Comment by prckent** [comment]
@ye-luo OK now?

**Comment by ye-luo** [comment]
@prckent Yes. It is good now.
@markdewing github doesn't send notification when you force push. Please leave a note when you did it.

**Pull request merged**